react-basic-practice/src/contexts/PopupProvider.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext, useMemo, useState } from 'react';
+
+const PopupContext = React.createContext();
+
+const usePopup = () => useContext(PopupContext);
+
+const PopupProvider = (props) => {
+  const { children } = props;
+  const [isSuccess, setIsSuccess] = useState(false);
+  const [openPopup, setOpenPopup] = useState(false);
+  const [message, setMessage] = useState('');
+
+  const valueContext = useMemo(() => ({
```

Comment:
Can you send one `set...` method instead of 3 `setOpenPopup, setIsSuccess, setMessage`.

Reply:
If I do the above, the code will be compact but it will be easy to get confused when calling, so can I keep the 3 separate methods as above?